### PR TITLE
Embedding-CSP: Intersection of a list of policies.

### PR DESCRIPTION
**Pull request by Sun77789**
To be committed after #163

### CODE REVIEW

**Comment by Sun77789** [comment]
To be committed after https://github.com/w3c/webappsec-csp/pull/163

File: embedded/index.src.html [hunk 1]
```diff
@@ -753,6 +753,64 @@ <h4 id="subsume-se" algorithm>
...
   5.  Return "`Does Not Subsume`".
 
+  <h4 id="intersection-policy-list" algorithm>
+    What is an intersection of a |policy list| for an
+    <a for="request">origin</a>?
+  </h4>
+
+  <a>Policy</a> (|intersection|) is said to be an 
+  <dfn for="policy">intersection</dfn> of list of <a>policy</a> objects
+  (|policy list|) if each policy in |policy list| <a>subsumes</a> |intersection|.
```
Comment (mikewest) [inline embedded/index.src.html]: How about: "`The <dfn for="policy">intersection</dfn> of a <a>list</a> of <a>policy</a> objects is a single <a>policy</a> which is <a>subsumed</a> by each item in the list.`"?

File: embedded/index.src.html [hunk 2]
```diff
@@ -753,6 +753,64 @@ <h4 id="subsume-se" algorithm>
...
+  In other words, |intersection| alone has the same effect as
+  [[csp3#multiple-policies]].
```
Comment (mikewest) [inline embedded/index.src.html]: How about: "`In other words, the <a for="policy">intersection</a> of of multiple <a>policy</a> objects represents the effective policy created when each of those policy object is applied to a single context (see [[csp3#multiple-policies]] for more detail).`"

**Comment by Sun77789** [comment]
Updated!

**Comment by mikewest** [comment]
LGTM. Will land it after the other one.

**Comment by mikewest** [comment]
Can you resolve the merge conflict on this patch? Then I'm happy to land it for you.

**Comment by Sun77789** [comment]
Rebased!

**Pull request merged**